contracts/utils/math/Math.sol

```diff
@@ -149,7 +146,7 @@ library Math {
 
             // Make sure the result is less than 2^256. Also prevents denominator == 0.
             if (denominator <= prod1) {
-                revert MathOverflowedMulDiv();
+                Panic.panic(denominator == 0 ? Panic.DIVISION_BY_ZERO : Panic.ARITHMETIC_UNDER_OR_OVERFLOW);
```

Comment:
We shall reflect this in the changesets. It's not a breaking change because we're not guaranteeing backwards compatibility for error interfaces but it'd be cool if we add a changelog entry